modules/text2vec-openai/clients/vectorizer.go

```diff
@@ -50,6 +51,10 @@ type openAIApiError struct {
 }
 
 func buildUrl(config ent.VectorizationConfig) (string, error) {
+	if host, ok := os.LookupEnv("OPENAI_BASE_URL"); ok {
```

Comment:
also here in `text2vec-openai` module, please move the lookup logic to [module.go](https://github.com/weaviate/weaviate/blob/stable/v1.19/modules/text2vec-openai/module.go#L105-L108) and pass the `openAIBaseURL` to client

Reply:
Would be great also to add this setting to [qna-openai](https://github.com/weaviate/weaviate/blob/stable/v1.19/modules/qna-openai/module.go#L135-L138) module

Reply:
I can replicate it there once i get the implementation here acceptable.  Pushed some changes, let me know what you think